src/components/Insights/tickets/common/ScalingIssueCommon/index.tsx

```diff
@@ -0,0 +1,72 @@
+import { trimEndpointScheme } from "../../../../../utils/trimEndpointScheme";
+import * as s from "./styles";
+import { ScalingIssueCommonProps } from "./types";
+import { getDurationString } from "../../../../../utils/getDurationString";
+
+export const ScalingIssueAffectedEndpoints = (
```

Comment:
pls place each of the component into separate folder

Reply:
done